src/main/resources/api.yaml

```diff
@@ -207,3 +237,21 @@ definitions:
     required:
       - enrollmentPeriod
       - durationPeriod
+  SprintSummaryDto:
+    type: object
+    properties:
+      participantAvailabilityMapping:
```

Comment:
robi się z tego mapa <String, Long> (a userId chyba już nie jest stringiem?)
wydaje mi się że lepsza byłaby lista { memberId, time1, time2, notes }

Reply:
Nie ogarniam tego obiektu - lista obiektów zawierających to `AvailabilityDto`  z issue 66 oraz COŚ identyfikującego użytkownika (albo samo id i potem osobne GETy, albo całe `UserDto` czy coś tam)?

Reply:
no okej, trochę się pozmieniało od tego czasu
{ AvailabilityDto, userId} <- o takie coś mi chodziło 
Listę użytkowników z ich danymi można dostać z tej samej metody, która pobiera skład projektu (i można ten skład trzymać sobie w storze + dorobić tam jakąś mapę {userId: MemberDto} żeby łatwiej się wyciągało dane konkretnego usera

Reply:
Jak `AvailabilityDto` dostanie `userId` to problem rozwiązany?

Reply:
Jedyna różnica jest taka, że nie będzie tego wrappera w postacji UserAvailabilityDto.

Reply:
dodałem pole `userId`